stable/kibana/templates/deployment.yaml

```diff
@@ -52,11 +52,16 @@ spec:
         - name: {{ template "kibana.fullname" . }}-importscript
           mountPath: "/tmp/dashboardImport.sh"
           subPath: dashboardImport.sh
+        {{- if .Values.configMap.override }}
```

Comment:
can we outdent all the `if` / `else` statements here to make this more readable please? (if you don't mind outdenting the `range` 's  while you are in here that would be helpful)